packages/core/src/runStream.js

```diff
@@ -1,4 +1,24 @@
-import { curry3 } from '@most/prelude'
+import { curry3, curry4 } from '@most/prelude'
+import RelativeSink from './sink/RelativeSink'
+import { schedulerRelativeTo } from '@most/scheduler'
 
+// Run a Stream, sending all its events to the
+// provided Sink.
 export const runStream = curry3((sink, scheduler, stream) =>
     stream.run(sink, scheduler))
+
+// Run a stream with its own localized clock
+// This transforms time from the provided scheduler's clock to a stream-local
+// clock (which starts at 0), and then *back* to the scheduler's clock before
+// propagating events to sink.  IOW, stream.run will see local times, and sink
```

Comment:
"IOW" is _slang_ for in other words. Google sees it as Isle of Wright. Wikipedia lists:

- Isle of Wight (frequently as IoW)
- The German abbreviation for the Leibniz Institute for Baltic Sea Research
- ISO 639-3 code of the Iowa-Oto language

https://en.wikipedia.org/wiki/IOW

I suggest that you only use official acronyms, so I think you should change it to "In other words, ..."

Reply:
Thanks! Fixed in 034e260